create matrix.js

```diff
@@ -1,3 +1,26 @@
+// This function is called by method string.replace() to correctly escape LaTex characters
+function myReplacer(match){
+  
+  // These chars are escaped prepending a backslash
+ if (match=="&" || match=="%" || match=="$" || match=="#" || match=="_" || match =="{" || match =="}")
```

Comment:
what about using a dictionary like
`escapeCharsDict={"&":"\&",
"%":"\%"
}`
and retrieving a substitution with `escpaeCharsDict["&"]` ?